configs/sst_cs_system_management-sys-management.yaml

```diff
@@ -123,4 +123,3 @@ data:
 
   labels:
   - eln
-  - c9s
```

Comment:
The gpm packages should be removed from this file.

Reply:
thanks for noticing it, I forgot to commit my changes - corrected in the latest commit.